invoker.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
   s.add_dependency("formatador", "~> 0.2")
   s.add_dependency("eventmachine", "~> 1.0.4")
   s.add_dependency("em-proxy", "~> 0.1")
-  s.add_dependency("rubydns", "~> 0.8.5")
+  s.add_dependency("rubydns", "> 0.8.5")
   s.add_dependency("uuid", "~> 2.3")
```

Comment:
I don't think invoker quite supports anything greater than `0.8.5`. IIRC rubydns was migrated from Eventmachine to Celluloid whereas bits of invoker still depends on Eventmachine.